draft-ietf-sipcore-sip-push.xml

```diff
@@ -387,12 +387,53 @@ align="center"><artwork>
           </t>
           <t>
             NOTE: If the UA uses a non-push mechanism to wake and send binding-refresh REGISTER requests, such REGISTER
-            requests will update the binding expiration timer, and the proxy does not need to request that a push notification is sent 
-            to the UA in order to wake the UA. The proxy will still request that a push notification is sent to the UA when the proxy receives 
+            requests will update the binding expiration timer, and the proxy does not need to request that a push notification be sent 
+            to the UA in order to wake the UA. The proxy will still request that a push notification be sent to the UA when the proxy receives 
             a SIP request addressed towards the UA (<xref target="section.proxy.req.oth"></xref>). This allows the UA to e.g., use timers for sending 
             binding-refresh REGISTER requests, but to be suspended (in order to save battery resources, etc.) between sending 
             the REGISTER requests and use push notification to wake the UA to process incoming calls.
           </t>      
+              <figure title="SIP REGISTER Example" anchor="fig-sip-pn-reg"
+align="center"><artwork>
+<![CDATA[
+
+    Example of a SIP REGISTER request including a 'sip.pnsreg'
+    media feature tag:
+
+    REGISTER sip:alice@example.com SIP/2.0
+    Via: SIP/2.0/TCP alicemobile.example.com:5060;branch=z9hG4bKnashds7
+    Max-Forwards: 70
+    To: Alice <sip:alice@example.com>
+    From: Alice <sip:alice@example.com>;tag=456248
+    Call-ID: 843817637684230@998sdasdh09
+    CSeq: 1826 REGISTER
+    Contact: <sip:alice@alicemobile.example.com;
+      pn-provider=acme;
+      pn-param=acme-param;
+      pn-prid=ZTY4ZDJlMzODE1NmUgKi0K>;
+      +sip.pnsreg
+    Expires: 7200
+    Content-Length: 0
+
+    Example of a SIP REGISTER response including a 'sip.pnsreg'
+    media feature tag and a 'sip.pnsreq' feature-capability indicator:
+
+    SIP/2.0 200 OK
+    Via: SIP/2.0/TCP alicemobile.example.com:5060;branch=z9hG4bKnashds7
+    To: Alice <sip:alice@example.com>;tag=123987
+    From: Alice <sip:alice@example.com>;tag=456248
+    Call-ID: 843817637684230@998sdasdh09
+    CSeq: 1826 REGISTER
+    Contact: <sip:alice@alicemobile.example.com;
+      pn-provider=acme;
+      pn-param=acme-param;
+      pn-prid=ZTY4ZDJlMzODE1NmUgKi0K>;
+      +sip.pnsreg
+    Feature-Caps: *;+sip.pnsreg
```

Comment:
According to the last paragraph of section 5.6.1.1, "the proxy SHOULD insert a 'sip.pnsreg' feature-capability indicator with an indicator value...".
So, shouldn't the example be like '+sip.pnsreg="121"' (a value bigger than 120).

Also, according to section 5.4, "These feature-capability indicators MUST only be inserted in a Feature-Caps header field that also contains a 'sip.pns' feature-capability indicator".
So (and according to other paragraphs), the Feature-Caps should also include '+sip.pns="acme"'.

Reply:
You are correct on both issues. I will fix it.